django-stubs/urls/conf.pyi

```diff
@@ -10,6 +10,7 @@ from ..conf.urls import IncludedURLConf
 from ..http.response import HttpResponseBase
 
 _URLConf: TypeAlias = str | ModuleType | Sequence[_AnyURL]
+_ResponseType: TypeAlias = HttpResponseBase | Coroutine[Any, Any, HttpResponseBase] | Coroutine[Any, Any, None]
```

Comment:
Sorry, I don't think that adding `Coroutine[Any, Any, None]` is correct.

In your case:

```python
async def async_view(): 
    return HttpResponse(status=200)
```

it does not return `None`, it returns `HttpResponse`.

The correct fix is:

```python
async def async_view() -> HttpResponse: 
    return HttpResponse(status=200)
```

Reply:
Yes, I agree with you!)

I changed None to HttpResponse